pkg/tanka/parse.go

```diff
@@ -51,8 +51,19 @@ func parse(baseDir string, opts *options) (*ParseResult, error) {
 	}, nil
 }
 
-// Eval returns the raw evaluated Jsonnet and the parsed env used for evaluation
-func Eval(baseDir string, opts *options) (raw map[string]interface{}, env *v1alpha1.Config, err error) {
+// Eval returns the raw evaluated Jsonnet output (without any transformations)
+func Eval(baseDir string, mods ...Modifier) (raw map[string]interface{}, err error) {
```

Comment:
This is even better than before, where an exported function was using an unexported type :)